freegames/overlap.py

```diff
@@ -0,0 +1,98 @@
+"""Overlap, classic arcade game.
+
+Exercises
+
+1. Change the shape.
+2. Change the color.
+3. Change the speed of rectangle.
+4. Change the a diminishing amount.
+5. Change the the direction of the path(ex.horizontal orientation).
+6. Change thr speed of frame.
+"""
+
+from turtle import *
+from freegames import vector
+
+rectangle = vector(0, 0)
+move = vector(0, 5)
+top = {0: vector(0, 0)}
+top_size = {0: 300, 1: 200}
```

Comment:
Should these be dictionaries or lists? Seems like only the last value is ever deleted.